openhands/controller/agent_controller.py

```diff
@@ -72,6 +72,7 @@
 TRAFFIC_CONTROL_REMINDER = (
     "Please click on resume button if you'd like to continue, or start a new task."
 )
+ERROR_ACTION_NOT_EXECUTED = 'The action has not been executed. This likely occurred because the runtime system crashed and was rebooted due to resource limitations. Be aware that any dependencies, environment variables, or other system state previously set up by the agent may have been lost after the reboot process.'
```

Comment:
```suggestion
ERROR_ACTION_NOT_EXECUTED = 'The action has not been executed. This may have occurred because the user pressed the stop button, or because the runtime system crashed and restarted due to resource constraints. Any previously established system state, dependencies, or environment variables may have been lost.'
```

@enyst how about this if we consider about "STOP" button?

Reply:
Yeah, i imagine we won't run into a lot of these issue "on local" unless user pressed stop!